docs/services/navigation/_index.md

```diff
@@ -554,3 +554,137 @@ obstacles = await my_nav.get_obstacles()
 
 {{% /tab %}}
 {{< /tabs >}}
+
+## Concepts
+
+The following concepts are important to understand when utilizing the navigation service.
+Each concept is a type of relative or absolute measurement, taken by a [movement sensor](/components/movement-sensor/), which can then be utilized by your robot to navigate across space.
+
+Here's how to make use of the following types of measurements:
+
+- [Compass Heading](/services/navigation/#compass-heading)
+- [Orientation](/services/navigation/#orientation)
+- [Angular Velocity](/services/navigation/#angular-velocity)
+- [Position](/services/navigation/#position)
+- [Linear Acceleration](/services/navigation/#linear-acceleration)
+- [Linear Velocity](/services/navigation/#linear-velocity)
+
+### Compass Heading
+
+The following {{< glossary_tooltip term_id="model" text="models" >}} of [movement sensor](/components/movement-sensor/) take compass heading measurements:
+
+- [gps-nmea](/components/movement-sensor/gps/gps-nmea/)
+- [gps-nmea-rtk-pmtk](/components/movement-sensor/gps/gps-nmea-rtk-pmtk/)
+- [gps-nmea-rtk-serial](/components/movement-sensor/gps/gps-nmea-rtk-serial/)
+- [imu-wit](/components/movement-sensor/imu/imu-wit/)
+
+An example of a `Compass Heading` reading:
+
+``` go
+gps, err := gps.CompassHeading(context.Background, nil)
+gps.CompassHeading{25}
+```
+
+If you want to read compass headings, [configure a capable movement sensor](/components/movement-sensor/#configuration) on your robot.
+The movement sensor API's [`GetCompassHeading()`](/components/movement-sensor/#getcompassheading) takes compass heading readings.
+Use compass heading readings to determine the *bearing* of your robot, or, the [cardinal direction](https://en.wikipedia.org/wiki/Cardinal_direction) that your robot is facing.
+
+### Orientation
+
+The following {{< glossary_tooltip term_id="model" text="models" >}} of [movement sensor](/components/movement-sensor/) take orientation measurements:
+
+- [imu-wit](/components/movement-sensor/imu/imu-wit/)
+- [imu-vectornav](/components/movement-sensor/imu/imu-vectornav/)
+
+An example of an `Orientation` reading:
+
+``` golang
+orientation, err := imuwit.Orientation(context.Background, nil)
+imu.Orientation{"_type":"quat","i":0,"j":0,"k":0,"r":1}
```

Comment:
```suggestion
// orientation is a OrientationVector{OX: 0, OY:0, OZ:1, Theta:0}
orientation, err := imuwit.Orientation(context.Background, nil)
```

Reply:
added

Reply:
This makes sense for showign users the actual data type, thanks again